src/mixins/nft.js

```diff
@@ -371,10 +371,10 @@ export default {
           const historyInDB = data.list;
           const eventMap = new Map();
           historyOnChain.forEach(e => {
-            eventMap.set(`${e.txHash}-${e.nftId}`, e);
+            eventMap.set(`${e.txHash}-${e.nftId}-${e.event}`, e);
```

Comment:
Do we really want to key each `event`, or do we just want `message` keys?
If its just `message` maybe `msg_index` will do

Reply:
So, the key would be `{txHash}-{msgIndex}`?

Reply:
> In what case do we have `mint_nft` and `new_class` in a same transaction? It sounds impossible since we don't have a `class_id` for `mint_nft` if `new_class` is not passed as a transaction first..

Sorry for typo, should be `mint_nft` and `/cosmos.nft.v1beta1.MsgSend`, not `new_class`.

Reply:
Oh I looked again and the event api only returns event name not msg_index
So current `event` key is good